securedrop/journalist.py

```diff
@@ -188,6 +184,9 @@ def admin_add_user():
                 flash('There was an error with the autogenerated password. '
                       'User not created. Please try again.', 'error')
                 form_valid = False
+            except InvalidUsernameException as e:
+                form_valid = False
+                flash('Invalid username: ' + str(e), "error")
```

Comment:
We should flash the same informative message we do on the command line: "Username blah is too short. Minimum characters: `Journalist.MIN_USERNAME_LEN`"